src/utils/interface.js

```diff
@@ -76,13 +85,7 @@ export const helpDialogue = () => {
 };
 
 export const messageCenter = () => {
-  const messageCenter = blessed.log({
-    border: false,
-    bottom: 2,
-    height: 5,
-    width: '100%',
-  });
-
+  const messageCenter = blessed.log(themeCommon(2, 5));
```

Comment:
With the comment above we can keep this as

```
const blessedObject = blessed.element({
	...baseTheme,
	bottom: 2,
	height: 5,
});
```

Let me know what you think! Thanks!

Reply:
Actually, I though that `bottom` and `height` also noticed as a similar, so I localize using a function which passing `bottom` and `height` as a params. Since they have different value.

That should be simply if only using an object. Thanks for suggestion @jwu910 !. I'll fix then.

Reply:
That makes sense! Don't worry, I just think that the flow of understanding the code for future maintainability may be more straight forward if we just use a plain object for now.